cli/module_generate.go

```diff
@@ -186,8 +186,23 @@ func (c *viamClient) generateModuleAction(cCtx *cli.Context, args generateModule
 
 // Prompt the user for information regarding the module they want to create
 // returns the common.ModuleInputs struct that contains the information the user entered.
-func promptUser() (*common.ModuleInputs, error) {
-	var newModule common.ModuleInputs
+func promptUser(module *common.ModuleInputs) error {
+	titleCaser := cases.Title(language.Und)
+	resourceOptions := []huh.Option[string]{}
+	for _, resource := range common.Resources {
```

Comment:
This is definitely a flyby request (and I shoulda caught this on Julie's PR a while back, my bad!), but I'd love to rename `common` as a package. It's really generic which means 1) it doesn't really indicate anything about what the package does, and 2) it risks namespace collision if anyone else has a `common` package. I think we could probably just leave it as `cli` and then make the values in it private (since I don't think we expect people to use them anyways), or could rename it to `modulegen` or something like that if we _do_ want to keep it distinct.

Reply:
The structs are used in another packages we have (the `scripts` package), so I can rename it to `modulegen`!